Assets/Scripts/UI/SkillBar.cs

```diff
@@ -0,0 +1,28 @@
+﻿using UnityEngine;
+using System.Collections;
+using UnityEngine.UI;
+
+public class SkillBar : MonoBehaviour 
+{
+	[SerializeField] Stat _stat = 0;
+	[SerializeField] float _statScaleMod = 200f;
+
+	RectTransform _uiBarRectFront = null;
+	RectTransform _uiBarRectBack = null;
+
+	PlayerStats _playerStats = null;
+
+	void Awake()
+	{
+		_uiBarRectBack = GetComponent<RectTransform>();
+		_uiBarRectFront = transform.GetChild(0).GetComponent<RectTransform>();
```

Comment:
You're using a member/property named `transform` here but I don't see where it's coming from. Is this one of the old deprecated automatic properties?